### PR TITLE
adding cron task for a partial cleaning repl_monitor table

### PR DESCRIPTION
**Pull request by kkonnov**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -70,3 +70,12 @@ repmgr_promote_command: repmgr standby promote -f {{ repmgr_conf }}
 repmgr_follow_command: repmgr standby follow -f {{ repmgr_conf }} -W
 repmgr_monitor_interval_secs: 2
 repmgr_retry_promote_interval_secs: 300
+
+# Cron variables:
```
Comment (UnderGreen) [inline defaults/main.yml:74]: It's wrong. Need a normal prefix for variabes, like 'repmgr_'. And need a implementation with dict.
https://github.com/alexey-medvedchikov/ansible-barman/blob/master/defaults/main.yml

File: tasks/main.yml [hunk 2]
```diff
@@ -10,3 +10,6 @@
   when: repmgr_bootstrap|bool
   tags: [repmgr, repmgr-bootstrap]
 
+- include: repl_monitor_cron.yml
```
Comment (UnderGreen) [inline tasks/main.yml]: Maybe `cron_jobs.yml`? Because we can have more than clear monitor table job.

File: tasks/repl_monitor_cron.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+ - name: Removing old records in the repl_monitor table
+   cron:
+     name: "Removing repl_monitor records older than 4 days"
+     minute: "{{ cron_minute }}"
+     hour: "{{ cron_hour }}"
+     job: "{{ cron_job }}"
```
Comment (UnderGreen) [inline tasks/repl_monitor_cron.yml]: https://github.com/alexey-medvedchikov/ansible-barman/blob/master/tasks/configure.yml

File: defaults/main.yml (hunk 1 above)
Comment (kkonnov) [inline defaults/main.yml:74]: Changed. Could you please check.

File: defaults/main.yml [hunk 4]
```diff
@@ -70,3 +70,18 @@ repmgr_promote_command: repmgr standby promote -f {{ repmgr_conf }}
...
+
+repmgr_cron: []
+
+#  hour: 4
```
Comment (UnderGreen) [inline defaults/main.yml]: Before this line need a '# repmgr_cron:'

Comment (UnderGreen) [inline defaults/main.yml]: And fix indentation, please

File: tasks/cron_jobs.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+ - name: Removing old records in the repl_monitor table
```
Comment (UnderGreen) [inline tasks/cron_jobs.yml]: Why here is an indentation?

File: tasks/cron_jobs.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    job: "psql -U postgres repmgr -c \"DELETE * FROM repmgr_test.repl_monitor WHERE last_monitor_time < current_timestamp - interval '4d'\""
+    state: present
+    disabled: "{{ item.disabled }}"
+    minute: "{{ item.minute }}"
+    hour: "{{ item.hour }}"
+    day: "{{ item.day }}"
+    month: "{{ item.month }}"
+    weekday: "{{ item.weekday }}"
+
+with_items: "{{ repmgr_cron }}"
```
Comment (UnderGreen) [inline tasks/cron_jobs.yml]: Please use check syntax before.

**Comment by kkonnov** [comment]
please check

**Pull request closed without merging**